addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,41 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export default class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+    }
+
+    async openRecord(record, mode) {
+        if (this.props.context.is_from_registration_desk) {
+            const barcode = record.data.barcode
+            const eventId = record.data.event_id[0]
+
+            const result = await this.orm.call("event.registration", "register_attendee", [], {
+                barcode: barcode,
+                event_id: eventId,
+            });
+
+            this.dialog.add(
+                EventRegistrationSummaryDialog,
+                {
+                    registration: result,
+                    model: this.model
+                }
+            )
```

Comment:
General comment again. Are we sure all actions possible to do on the dialog will correctly happen on both views? Please double check if not done already :)

Reply:
Hello @Odoonan 

It is working fine functionally.. can you please elaborate your concerns though. (I checked technically too, it seems fine)

Thanks :smile: